core/templates/dev/head/domain/skill/ConceptCardBackendApiService.ts

```diff
@@ -101,16 +101,16 @@ oppia.factory('ConceptCardBackendApiService', [
        * Returns whether the given concept card is stored within the local data
        * cache or if it needs to be retrieved from the backend upon a laod.
        */
-      isCached: function(skillId) {
-        return _isCached(skillId);
+      isCached: function(skillIds) {
+        return _isCached(skillIds);
       },
 
       /**
        * Replaces the current concept card in the cache given by the specified
        * skill ID with a new concept card object.
        */
-      cacheConceptCard: function(skillId, conceptCard) {
-        _conceptCardCache[skillId] = angular.copy(conceptCard);
+      cacheConceptCard: function(skillIds, conceptCard) {
```

Comment:
Are we caching multiple concept cards? The naming is a little confusing

Reply:
Done.

Reply:
Wait I didn't understand what changed here?

Reply:
Done.